New sections in the CSS reference article for the shadow and easing editors (#2798)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/css/reference.md

> 1. Start typing `coverage`, and then select **Show Coverage**.  The **Coverage** tool appears.

Reviewer: ```suggestion
1. Start typing `coverage`, and then select **Show Coverage**.  The **Coverage** tool appears:
```

>    Opening the **Coverage** tool from the Command Menu:
> 

Reviewer: ```suggestion
```

   ![Opening the Coverage tool from the Command Menu](./reference-images/css-console-command-menu-coverage.png)

   The **Coverage** tool:

>    ![The Coverage tool](./reference-images/css-console-qs-coverage-empty.png)

Reviewer: ```
1. Start typing `coverage`, and then select **Show Coverage**:

   ![Opening the Coverage tool from the Command Menu](./reference-images/css-console-command-menu-coverage.png)

   The **Coverage** tool appears:

   ![The Coverage tool](./reference-images/css-console-qs-coverage-empty.png)
```
move pngs vs text, & tighten

By default, adding a style rule by clicking on **New Style Rule** inserts the new rule beneath the **element.style** rule in the `inspector-stylesheet` stylesheet.

> To add a style rule in a specific location of the **Styles** pane instead:

Reviewer: No change proposed here; a style comment/idea:
* When need to envision & put focus on selecting/clicking the little tab in the tab-set, say "tab".
* When putting focus on content that's inside the entire tab ("tab" in the encompassing, broad sense), say "pane".

Depends on what you have in mind: the little tab to click, vs. the entire tab/panel that's accessed via the little tab.

There are more keyboard shortcuts to change the angle value.  Find out more in the [Styles pane keyboard shortcuts](../shortcuts/index.md#styles-pane-keyboard-shortcuts).

<!-- ====================================================================== -->
## Change box and text shadows with the Shadow Editor

Use the **Shadow Editor** to change the value of the `box-shadow` or `text-shadow` CSS property on an HTML element:

1. [Select an element](#select-an-element) with a `box-shadow` or `text-shadow` declaration.

   For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, and then select the `<div class="demos">` element in the **Elements** tool.

1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.

>    The **Shadow Editor** appears:

Reviewer: ```suggestion
   The **Shadow Editor** opens:
```

   ![The Shadow Editor](./reference-images/shadow-editor.png)

> 1. Change the shadow properties:

Reviewer: ```suggestion
1. Change the shadow properties, as follows:
```

>    * Type (only for box-shadow). Pick **Outset** or **Inset**.
>    * X and Y offsets. Drag the blue dot or specify values.
>    * Blur. Drag the slider or specify a value.
>    * Spread (only for box-shadow). Drag the slider or specify a value.

Reviewer: ```suggestion
   | Property | How to change |
   |---|---|
   | **Type** | Select **Outset** or **Inset**.  Only for `box-shadow`. |
   | **X offset** | Drag the blue dot, or specify a value. |
   | **Y offset** | Drag the blue dot, or specify a value. |
   | **Blur** | Drag the slider or specify a value. |
   | **Spread** | Drag the slider or specify a value.  Only for `box-shadow`. |
   <!-- todo: explain the meaning of each property, not only how to change the value.  change column heading to:
   | Property | Description |
   | Property | Description and how to change |
   -->
```
possible todo: explain the meaning of each property, not only how to change the value; and change column heading to eg:
```
   | Property | Description |
   | Property | Description and how to change |
```

There are more keyboard shortcuts to change the angle value.  Find out more in the [Styles pane keyboard shortcuts](../shortcuts/index.md#styles-pane-keyboard-shortcuts).

<!-- ====================================================================== -->
## Change box and text shadows with the Shadow Editor

Use the **Shadow Editor** to change the value of the `box-shadow` or `text-shadow` CSS property on an HTML element:

1. [Select an element](#select-an-element) with a `box-shadow` or `text-shadow` declaration.

>    For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, and then select the `<div class="demos">` element in the **Elements** tool.

Reviewer: ```suggestion
   For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, right-click the page and select **Inspect** to open DevTools, and then in the **Elements** tool, select the `<div class="demos">` element.
```

> 1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.

Reviewer: ```suggestion
1. In the **Styles** pane, in the `.demos` CSS rule, find the `box-shadow` declaration, and then click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.
```
in order of UI eye-tracking/ finding of controls

   The **Shadow Editor** opens:

   ![The Shadow Editor](./reference-images/shadow-editor.png)

1. Change the shadow properties, as follows:

   | Property | How to change |
   |---|---|
   | **Type** | Select **Outset** or **Inset**.  Only for `box-shadow`. |

>    | **X offset** | Drag the blue dot, or specify a value. |
>    | **Y offset** | Drag the blue dot, or specify a value. |
>    | **Blur** | Drag the slider or specify a value. |
>    | **Spread** | Drag the slider or specify a value.  Only for `box-shadow`. |

Reviewer: ```suggestion
   | **X offset** | Specify a value in the text box, or drag the blue dot. |
   | **Y offset** | Specify a value in the text box, or drag the blue dot. |
   | **Blur** | Specify a value in the text box, or drag the slider. |
   | **Spread** | Drag the slider or specify a value.  Only for `box-shadow`. |
```

   <!-- todo: explain the meaning of each property, not only how to change the value.  change column heading to:
   | Property | Description |
   | Property | Description and how to change |
   -->

>    The changes are applied to the element in the page in real time:

Reviewer: ```suggestion
   The changes are applied to the element in the rendered webpage in real time:
```

   ![The effect of changes made in the Shadow Editor](./reference-images/shadow-editor-changes.png)

<!-- ====================================================================== -->
## Edit animation and transition timings with the Easing Editor

Use the **Easing Editor** to change the value of the [animation-timing-function](https://developer.mozilla.org/docs/Web/CSS/animation-timing-function) or [transition-timing-function](https://developer.mozilla.org/docs/Web/CSS/transition-timing-function) property on an HTML element:

1. [Select an element](#select-an-element) that has a CSS animation or transition applied.

>    For example, open the [animated property demo page](https://microsoftedge.github.io/Demos/devtools-animated-property-issue/) in a new tab or window, and then select the `<div class="spinner">` element in the **Elements** tool.

Reviewer: ```suggestion
   For example, open the [animated property demo page](https://microsoftedge.github.io/Demos/devtools-animated-property-issue/) in a new tab or window, right-click the page and select **Inspect** to open DevTools, and then select the `<div class="spinner">` element in the **Elements** tool.
```

>    The red and green boxes on this demo page are two different CSS animations. Both animations run with CSS, using the `animation` CSS property, and define an `ease-in-out` timing function.

Reviewer: ```suggestion
   The red and green boxes on this demo page are two different CSS animations. Both animations run with CSS by using the `animation` CSS property, and define an `ease-in-out` timing function.
```
b/c i had trouble parsing the comma-delimited list

1. In the **Styles** pane, find the `animation` declaration in the `.bad .spinner` CSS rule, and click the **Open cubic bezier editor** (![The cubic bezier editor button icon](./reference-images/easing-editor-icon.png)) button.

>    The **Easing Editor** appears:

Reviewer: ```suggestion
   The **Easing Editor** opens:
```
in some cases, i use 'appears', but this kind of editor popup goes well w/ "opens".

   ![The Easing Editor](./reference-images/easing-editor.png)

1. Change the properties of the animation timing function:

   * Choose one of the pre-defined timing functions from the list.
   * Create your own timing function by dragging the cubic-bezier curve handles.

>    The changes are applied to the element in the page in real time:

Reviewer: ```suggestion
   The changes are applied to the element in the rendered webpage in real time:
```

There are more keyboard shortcuts to change the angle value.  Find out more in the [Styles pane keyboard shortcuts](../shortcuts/index.md#styles-pane-keyboard-shortcuts).

<!-- ====================================================================== -->
## Change box and text shadows with the Shadow Editor

Use the **Shadow Editor** to change the value of the `box-shadow` or `text-shadow` CSS property on an HTML element:

1. [Select an element](#select-an-element) with a `box-shadow` or `text-shadow` declaration.

   For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, and then select the `<div class="demos">` element in the **Elements** tool.

> 1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** button (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)).

Reviewer: ```suggestion
1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.
```

   The **Shadow Editor** appears:

   ![The Shadow Editor](./reference-images/shadow-editor.png)

1. Change the shadow properties:

   * Click **Outset** or **Inset** to change the shadow type.
   * Click and drag the handle in the chart to change the shadow offset.
   * Use the **Blur** slider to change the amount of shadow blur.
   * Use the **Spread** slider to change the amount of shadow spread.

> 1. The changes are applied to the element in the page in real time:

Reviewer: ```suggestion
   The changes are applied to the element in the page in real time:
```
action/result format.  Actions are numbered, results are not

   ![The effect of changes made in the Shadow Editor](./reference-images/shadow-editor-changes.png)

<!-- ====================================================================== -->
## Edit animation and transition timings with the Easing Editor

> Use the **Easing Editor** to change the value of the `animation-timing-function` or `transition-timing-function` property on an HTML element:

Reviewer: ```suggestion
Use the **Easing Editor** to change the value of the [animation-timing-function](https://developer.mozilla.org/docs/Web/CSS/animation-timing-function) or [transition-timing-function](https://developer.mozilla.org/docs/Web/CSS/transition-timing-function) property on an HTML element:
```
unless not linking is by-design

1. [Select an element](#select-an-element) that has a CSS animation or transition applied.

   For example, open the [animated property demo page](https://microsoftedge.github.io/Demos/devtools-animated-property-issue/) in a new tab or window, and then select the `<div class="spinner">` element in the **Elements** tool.

1. In the **Styles** pane, find the `animation` declaration in the `.bad .spinner` CSS rule, and click the **Open cubic bezier editor** (![The cubic bezier editor button icon](./reference-images/easing-editor-icon.png)) button.

   The **Easing Editor** appears:

   ![The Easing Editor](./reference-images/easing-editor.png)

1. Change the properties of the animation timing function:

   * Choose one of the pre-defined timing functions from the list.
   * Create your own timing function by dragging the cubic-bezier curve handles.

> 1. The changes are applied to the element in the page in real time:

Reviewer: ```suggestion
   The changes are applied to the element in the page in real time:
```
a system-result step, not a user-action step, so no #
